app/helpers/pages_helper.rb

```diff
@@ -9,6 +9,10 @@ def page_nav_item(text, path, strict=true)
     end
   end
 
+  def serialize(data, field)
+    (data.stringify_keys[field.to_s] || {}).to_json.html_safe
+  end
```

Comment:
The only difference is that I'm not just doing the `||` for if there's no key match, I'm also doing it in case the value is anything blank. I think I'll add a blank check, cause I'm currently messing up the `""` case.